### DISCUSSION TITLE
Query about robust Aitchison in vegdist() and decostand() functions

### DISCUSSION THREAD

**Discussion by RachBioHaz** [vegandevs/vegan#566] [Q&A]
Hi, I'm new to analysis. I am looking to use the robust Aitchison distance for my dissimilarity matrix. I was looking at the arguments and I wanted to know if I interpreted them correctly.

I was working through an example that used the "hellinger" to standardise the data, coupled with "euclidian" for distance:
hell_di <- vegdist(decostand(mite, method = "hellinger"), method = "euclidean")

I tried to do a similar thing with "rclr" and "robust.aitchison" and it didn't work. Not sure I've understood the documentation correctly, but am I right in thinking that I do not need to standardise the data in decostand if I am using "robust.aitchison" in vegdist?

Let me know if I am completely wrong in my interpretation. 
Cheers

**Comment by antagomir** [comment]
You are correct. 

The following two are equivalent (and x>0 always for clr):

`vegdist(decostand(x, method = "clr"), method = "euclidean")`
`vegdist(x, method = "aitchison")`


Similar thing applies to `rclr` and `robust.aitchison`.

**Reply by RachBioHaz** [reply]
Thank you
